dev/archery/archery/docker/cli.py

```diff
@@ -122,8 +122,8 @@ def docker_pull(obj, image, *, pull_leaf, ignore_pull_failures):
                      ignore_pull_failures=ignore_pull_failures)
     except UndefinedImage as e:
         raise click.ClickException(
-            "There is no service/image defined in docker-compose.yml with "
-            "name: {}".format(str(e))
+            f"There is no service/image defined in docker-compose.yml with "
```

Comment:
`f` is not needed here

Reply:
Resolved in [2c7c552](https://github.com/apache/arrow/pull/45629/commits/2c7c552cf2119490867016bdce97578af1f1481b).